VAMobile/src/screens/BenefitsScreen/ClaimsScreen/AppealDetailsScreen/AppealStatus/AppealStatus.test.tsx

```diff
@@ -35,10 +33,6 @@ context('AppealStatus', () => {
     component = render(<AppealStatus {...props} />, {
       preloadedState: {
         ...InitialState,
-        personalInformation: {
-          ...InitialState.personalInformation,
-          profile: {} as UserDataProfile,
-        },
       },
     })
```

Comment:
Same as my comment above, `preloadedState` is no longer needed, since`InitialState` will already be set by default

Reply:
This one was also left behind